ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/css/formcontainerview.css

```diff
@@ -16,4 +16,12 @@
 
 [data-cmp-visible="false"] {
     display: none;
+}
+
+[data-cmp-longdescription-hidden=true] {
```

Comment:
we should have only one class for managing visible attribute.

Reply:
done